src/components/pages/Contacts/index.css

```diff
@@ -9,3 +9,18 @@
 #faded {
   opacity: 50%;
 }
+
+.stickitude {
```

Comment:
hahaha I meant to change the name of this class to just "sticky-toolbar" or something along those lines - but it probably will work fine being isolated to the contacts page